launcher/JavaCommon.cpp

```diff
@@ -73,7 +73,8 @@ void JavaCommon::javaWasOk(QWidget *parent, JavaCheckResult result)
     QString text;
     text += QObject::tr("Java test succeeded!<br />Platform reported: %1<br />Java version "
         "reported: %2<br />Java vendor "
-        "reported: %3<br />").arg(result.realPlatform, result.javaVersion.toString(), result.javaVendor);
+        "reported: %3<br />64-bit "
+        "reported: %4<br />").arg(result.realPlatform, result.javaVersion.toString(), result.javaVendor, result.is_64bit ? "Yes" : "No");
```

Comment:
I don't see how this is an improvement. `realPlatform` already contains the `os.arch` and displays it to the user

Reply:
That's true, but whether it's a 64-bit platform or not is not obvious for some architectures because they don't include a number (like 32-bit ARM).

Reply:
well in that case it might just be better to change the results to say
`Java bitness reported: 32bit` and `Java bitness reported: 64bit` rather than `64-bit reported: Yes` and `64-bit reported: No` which just doesn't sound right